### ISSUE TITLE
⌨ Voiceover separating single links into two items

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#14482]
### ⌨ Accessibility and Expected Behaviour

When reviewing your profile during the application process and navigating using the progress stepper, the current step is treated as two separate clickable elements instead of one, for the current step. 

They are both part of the same link for the current step, but Voiceover sees two elements. 

The first element is read as "Step current" and the second element is read as "Review your profile current step".

### 🕵️ Details

Operating sytem: IOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver

### 🙋‍♀️ Proposed Solution

Links for the same step should be treated as one element, not two.

**Comment by esizer** [comment]
Easily reproducable. Something to do with the mark up of our stepper. Best guess, is the fact we know there are issues with our hidden text.
